metron-interface/metron-config/src/app/sensors/sensor-parser-config/sensor-parser-config.component.ts

```diff
@@ -409,12 +403,17 @@ export class SensorParserConfigComponent implements OnInit {
     this.showFieldSchema = (pane === Pane.FIELDSCHEMA) ? visibilty : false;
     this.showRawJson = (pane ===  Pane.RAWJSON) ? visibilty : false;
     this.showThreatTriage = (pane ===  Pane.THREATTRIAGE) ? visibilty : false;
+    this.showStormSettings = (pane === Pane.STORMSETTINGS) ? visibilty : false;
   }
 
   onRawJsonChanged(): void {
     this.sensorFieldSchema.createFieldSchemaRows();
   }
 
+  onStormSettingsChanged(): void {
```

Comment:
Is this a debug artifact?

Reply:
Yes this was a mistake.  I think we should leave this function in the event we need it in the future but I will definitely remove this logging.